package.json

```diff
@@ -5,15 +5,14 @@
     "scripts": {
         "start": "cross-env PORT=3001 SKIP_PREFLIGHT_CHECK=true react-scripts start",
         "build": "react-scripts build",
-        "prepublishOnly": "rm -rf dist; npm run transpile; npm run transpile-tests",
+        "prepare": "husky install",
         "transpile-tests": "babel --presets @babel/preset-env --out-dir dist/tests-legacy/cucumber tests-legacy/cucumber",
-        "transpile": "tsc; node-sass -r src/stylesheets -o dist/stylesheets",
-        "test": "bash run-tests.sh",
-        "testsOnly": "bash run-tests.sh -s=true",
+        "transpile": "tsc && npm run copy-css",
+        "copy-css": "cp src/stylesheets/* dist/stylesheets/",
+        "test": "echo 'Tests are in the tests subfolder. Use Node 20 (`nvm use 20`) and run `npm i` && `npm test` there'",
```

Comment:
Can we add a `cd tests && nvm use 20 && npm i && npm test && cd -` instead of a directive? Or is this too fragile?

Reply:
Good idea, but with nvm there's too much logic in this command. Let's wait until we are on Node 20 for both app and tests